src/napari_imagej/_module_utils.py

```diff
@@ -581,18 +609,19 @@ def module_execute(
 
         # get output
         log_debug("Execution complete")
-        j_result = _module_output(module)
-        result = ij().py.from_java(j_result)
-        log_debug(f"Result = {result}")
```

Comment:
We shouldn't leave this out, but just printing the result is not the nicest output.

Can we do better?

Reply:
The output looks much nicer in 17e35df